src/Control/Optics/Linear/Internal.hs

```diff
@@ -48,3 +52,24 @@ _Right = Optical second
 
 traversed :: Traversable t => Traversal a b (t a) (t b)
 traversed = Optical wander
+
+over :: Optic_ LinearArrow a b s t -> (a ->. b) -> s ->. t
+over (Optical l) f = getLA (l (LA f))
+
+traverseOf :: Optic_ (LKleisli f) a b s t -> (a ->. f b) -> s ->. f t
+traverseOf (Optical l) f = runLKleisli (l (LKleisli f))
+
+get :: Optic_ (Kleisli (Const a)) a b s t -> s -> a
+get l = gets l P.id
+
+gets :: Optic_ (Kleisli (Const r)) a b s t -> (a -> r) -> s -> r
+gets (Optical l) f s = getConst (runKleisli (l (Kleisli (Const P.. f))) s)
+
+set :: Optic_ (->) a b s t -> b -> s -> t
+set (Optical l) x = l (const x)
+
+lengthOf :: Num r => Optic_ (Kleisli (Const (Sum r))) a b s t -> s -> r
+lengthOf l s = getSum (gets l (const (Sum 1)) s)
+
+traverseOf' :: Optic_ (Kleisli f) a b s t -> (a -> f b) -> s -> f t
```

Comment:
Let's make a note that this will be made redundant from a multiplicity polymorphism `traverseOf`.